String_Programs/Specialcharacter.py

```diff
@@ -0,0 +1,15 @@
+String = input('Enter the string : \n')
+count = 0
+value = ""
+String = String.lower()
+for i in String:
+    if i == '!' or i == '@' or i == '#' or i == '$' or i == '%' or i == '^' or i == '&'or i == '*' or i == '(' or i == i == '-' or i == '+' or i == '=' or i == '_' or i == ':' or i ==';' or i == '<' or i == '>' or i == '/' or i == '?' or i == '" ' :
```

Comment:
Are you looking only for these special chars? Understand the ASCII value concept: https://www.programiz.com/python-programming/examples/ascii-character Add these in kbits. 
You can find the range of ascii value of special char and compare just with those.
And even if you are looking to compare with these specific special char, then put all of these in the an array and then compare.